hooks/command

```diff
@@ -84,24 +115,46 @@ download_binary_and_run() {
   get_architecture || return 1
   local _arch="$RETVAL"
   local _executable="monorepo-diff-buildkite-plugin"
+  local _executable_version_file="monorepo-diff-buildkite-plugin.version"
```

Comment:
given that this value must be synchronized among 2 different functions it would make sense to make it a global variable instead